src/main/java/uk/gov/companieshouse/controller/AppealController.java

```diff
@@ -65,6 +72,11 @@ public AppealController(AppealService appealService) {
 
             return ResponseEntity.created(location).body(id);
 
+        } catch (AppealReasonException appealReasonException) {
+            LOGGER.error(appealReasonException.getMessage() +
+                " for company number {}, penalty reference {} and user id {}",
+                    companyId, penaltyReference, userId, appealReasonException);
+            return ResponseEntity.status(HttpStatus.BAD_REQUEST).build();
```

Comment:
We could move this and the validation into a seperate earlier try block to group them together and it might make it more obvious where this exception could come from and improve readability.